app/requirements.txt

```diff
@@ -9,7 +9,7 @@ MarkupSafe==1.1.1
 mccabe==0.6.1
 peewee==3.13.2
 psycopg2-binary==2.8.5
-git+git://github.com/jameshicks/pydigree@master
+#git+git://github.com/jameshicks/pydigree@master
```

Comment:
Why do you comment this line is not related to the database changes?

Reply:
This was not supposed to be committed. It was a local change that I needed to be able to load the data (the `pydigree` Python module did not install on my system).  I will revert this change.